using-mobile-extensions/adobe-campaignclassic/README.md

```diff
@@ -15,7 +15,7 @@
 ![Configuring the Campaign Classic extension](../../.gitbook/assets/acc-configure-extension.png)
 
 {% hint style="info" %}
-Trying to find your Campaign Classic registration or tracking endpoint URLs? You can retrieve this information in the Campaign Classic interface in the **Tools** &gt; **Advanced** &gt; **Deployment wizard** menu. The endpoint for push notifications is usually the same as the URL that is used for web forms and surveys.
+You can retrieve your Campaign Classic registration on tracking endpoint URLS in the Campaign Classic interface under the **Tools > Advanced > Deployment wizard** menu. The endpoint for push notifications is usually the same as the URL that is used for web forms and surveys.
```

Comment:
`registration on tracking endpoint URLS` should be reverted back to `registration or tracking endpoint URLS`